components/unique/Collection/Item/CollectionItem.vue

```diff
@@ -3,12 +3,7 @@
     <div class="columns is-centered">
       <div class="column is-half has-text-centered">
         <div class="container image is-128x128 mb-2">
-          <b-image
-            v-if="!isLoading"
-            :src="image"
-            :alt="name"
-            ratio="1by1"
-            rounded></b-image>
+          <BasicImage v-if="!isLoading" :src="image" :alt="name" rounded />
```

Comment:
How ration is maintained?

Reply:
For now we keep it 1b1 but It can be extended with custom prop later on.